frontend/src/components/ReadmeLogo.jsx

```diff
@@ -1,39 +1,6 @@
 import React from "react";
-import styled from "styled-components";
 
-const ReadmeLogo = () => <Readme>readme</Readme>;
-const ReadmeLogoBody = () => <ReadmeBody>readme</ReadmeBody>;
-
-const readmeIfy = (text, isBodyText = false) =>
-  text && (
-    <span>
-      {text
-        .split(/readme/)
-        .reduce(
-          (prev, current, i) =>
-            i
-              ? prev.concat(
-                  isBodyText ? (
-                    <ReadmeLogoBody key={current} />
-                  ) : (
-                    <ReadmeLogo key={current} />
-                  ),
-                  current
-                )
-              : [current],
-          []
-        )}
-    </span>
-  );
+const readmeIfy = (text) =>
```

Comment:
This could have a more generic name, especially now that readme is not even here